src/styles/_header.scss

```diff
@@ -0,0 +1,190 @@
+                    &::before {
+                        opacity: 1;
+                    }
+                }
+            }
+        }
+        
+    
+        &__menu { //mobile menu
+            > span {
+                display: block;
+                width: 50px;
+                height: 5px;
+                background-color: $white;
+                transition: all 300ms ease-in-out;
+                transform-origin: 10px 2px;
+    
+                &:not(:last-child){
+                    margin-bottom: 5px;
+                }
+            }
+    
+            &.open {
+                 
+                >span:first-child {
+                    transform: rotate(45deg);
+                }
+    
+                >span:nth-child(2) {
+                    opacity: 0;
+                }
+                
+                >span:last-child {
+                    transform: rotate(-45deg);
+                }
+            }
+        }
+    }
+
+    .overlay {
+        animation: fade-in 300ms ease-in-out 1 forwards;
+        position: fixed;
+        opacity: 1;
+        top: 3.75rem;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background-image: linear-gradient($darkBlue, transparent);
+        z-index: 1002;
+        overflow-y: hidden;
+    
+    }
+    .overlayOut {
+        animation: fade-out 500ms ease-in-out 1 forwards;
+        position: fixed;
+        top: 3.75rem;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background-image: linear-gradient($darkBlue, transparent);
+    }
+
+    .overlayId {
+        z-index: 10;
+    }
+
+    .menu_list {
+        background: $grayishBlue;
+        display: flex;
+        flex-direction: column;
+        margin-top: 2%;
+        margin-bottom: 2%;
+        text-align: center;
+        justify-content: center;
+        border-radius: 15px;
+        position: absolute;
+        width: calc(100% - 36rem);
+        left: 50%;
+        transform: translateX(-50%);
+        z-index: 10000;
+        top: 83px;
+        border: 1px solid black;
+    }
+    
+    a.menu_anchor {
+        margin-right: 0px;
+        display: flex;
+        color: white;
+        font-size: 1.7rem;
+        padding: 0.625rem;
+        justify-content: center;
+        text-decoration: none;
+
+        &:not(:last-child) {
+            border-bottom: 1px solid black;
+        }
+    }
+
+    .under_header {
+        font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+        display: flex;
+        position: relative;
+        top: -211px;
+        align-items: center;
+        justify-content: center;
+        background-color: $grayishBlue;
+        color: $white;
+        height: 174px;
+        z-index: -2;
+        border-top: solid 9px black;
+
+        @include breakpoint-down (medium) {
+            position: relative;
+            left: 50px;
+        }
+
+        @media screen and (max-width: 530px) {
```

Comment:
@Kamildeeal media screen brakpoints should be also defined as constants (I mean 530px etc)